Lib/test/test_typing.py

```diff
@@ -880,6 +880,11 @@ def test_cannot_be_called(self):
         with self.assertRaises(TypeError):
             Unpack()
 
+    def test_usage_with_kwargs(self):
+        Movie = TypedDict('Movie', {'name': str, 'year': int})
+        def foo(**kwargs: Unpack[Movie]): ...
+        self.assertEqual(repr(foo.__annotations__['kwargs']),
+                         f"typing.Unpack[<class '{__name__}.Movie'>]")
```

Comment:
Would be good if we can make this say `typing.Unpack[{__name__}.Movie]` instead. Is there already a helper for prettifying the repr() of classes?

Reply:
> Is there already a helper for prettifying the repr() of classes?

`typing._type_repr` should do it